src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx

```diff
@@ -0,0 +1,159 @@
+        this.props.store.mergedMutationDataIncludingUncalledFilteredByGene.forEach(
+            altData => {
+                const mutationType = {
+                    gene: altData[0].gene.hugoGeneSymbol,
+                    altered: 1,
+                    sequenced: 1,
+                    percentAltered: altData[0].mutationType,
+                    geneticTrackRuleSetParams: DEFAULT_RULESET_PARAMS,
+                    geneticTrackData: this.props.store.geneticTrackData.result
+                        ? this.props.store.geneticTrackData.result[
+                              altData[0].gene.hugoGeneSymbol
+                          ]
+                        : undefined,
+                };
+                if (mutationType) {
+                    alterationFrequencyData.push(mutationType);
+                }
+            }
+        );
+
+        this.props.store.mergedDiscreteCNADataFilteredByGene.forEach(
+            altData => {
+                const cna = {
+                    gene: altData[0].gene.hugoGeneSymbol,
+                    altered: 1,
+                    sequenced: 1,
+                    percentAltered: getCnaTypes(altData[0].alteration),
+                    geneticTrackRuleSetParams: DEFAULT_RULESET_PARAMS,
+                    geneticTrackData: this.props.store.geneticTrackData.result
+                        ? this.props.store.geneticTrackData.result[
+                              altData[0].gene.hugoGeneSymbol
+                          ]
+                        : undefined,
+                };
+                if (cna) {
+                    alterationFrequencyData.push(cna);
+                }
+            }
+        );
+
+        return alterationFrequencyData;
+    }
+
+    public render() {
+        if (!this.PathwayMapperComponent) {
+            return null;
+        }
+        return (
+            <div className="pathwayMapper">
+                <div
+                    data-test="pathwayMapperTabDiv"
+                    className="cBioMode"
+                    style={{ width: '99%' }}
+                >
+                    <Row>
+                        {/*
+                              // @ts-ignore */}
+                        <this.PathwayMapperComponent
+                            isCBioPortal={true}
+                            isCollaborative={false}
+                            genes={this.queryGenes}
+                            cBioAlterationData={this.alterationFrequencyData}
+                            tableComponent={this.renderTable}
+                            patientView={true}
+                            //message banner patch will be removed
+                            messageBanner={this.renderBanner}
+                        />
+                    </Row>
+                </div>
+            </div>
+        );
+    }
+
+    //will be removed when Banner is added
+    @autobind
+    private renderBanner() {
+        return null;
+    }
```

Comment:
I think we will have a separate PR to clean up this one. Ideally `messageBanner` should be an optional property of `PathwayMapperComponent`.